src/styles/block/body.scss

```diff
@@ -0,0 +1,18 @@
+body {
+  margin: auto 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  color: $color-black;
+  font-weight: 400;
+}
+
+p {
+  margin-top: 0;
+  margin-bottom: 0;
```

Comment:
Combine with `margin` property and don't style tags

Reply:
Let's to do it
https://github.com/mate-academy/layout_catalog/pull/3348/files#r1249309606